buildrunner/config/__init__.py

```diff
@@ -0,0 +1,353 @@
+"""
```

Comment:
maybe it's just the size of the code review talking to me but this move feels unnecessary, are we planning on adding more config in here at some point ?

Reply:
We had at least two files named "config.py", this was an attempt to simplify life and prevent confusion since everytime I was in here I had to look for the right config module at least twice.

Reply:
The move here was to get config easy to access in the rest of the app with `buildrunner.config.BuildRunnerConfig.get_instance()`

Reply:
oh, it looked like this __init__ was the only file in here at first, nevermind !